src/classes/myclass.cls

```diff
@@ -0,0 +1,9 @@
+public with sharing class myclass {
```

Comment:
It'd probably be worthwhile to follow the recommended [naming conventions for Apex classes](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_naming_conventions.htm).